spec/controllers/course/assessment/submission/answer/programming/programming_controller_spec.rb

```diff
@@ -39,6 +39,22 @@
           expect(answer.specific.files.count).to eq(2)
         end
       end
+
+      context 'when uploading new programming files with more than 1000 lines' do
+        # defining a file that has size more than 50KB
```

Comment:
Just to check, where is this validated? Are you checking for file size or line number here? the context description doesnt match.

Reply:
it's validated inside `app/models/course/assessment/answer/programming_file.rb`
I'm intending to check the size only (more than 50KB). I have fixed the context description properly.

Basically if you upload any file that contains more than 1000 lines or have size more than 50KB, the file won't be uploaded and error message will be spawn as such